nextcord/client.py

```diff
@@ -2652,3 +2655,40 @@ def parse_mentions(self, text: str) -> List[User]:
 
         it = filter(None, map(self.get_user, utils.parse_raw_mentions(text)))
         return utils._unique(it)
+
+    @overload
+    def get_interaction(self, data) -> Interaction:
+        ...
+
+    @overload
+    def get_interaction(self, data, *, cls: Type[Interaction]) -> Interaction:
+        ...
+
+    @overload
+    def get_interaction(self, data, *, cls: Type[InterT]) -> InterT:
+        ...
+
+    def get_interaction(
+        self, data, *, cls: Type[InterT] = Interaction
+    ) -> Union[Interaction, InterT]:
+        """Returns an interaction for a gateway event.
+
+        Parameters
+        ----------
+        data
+            The data direct from the gateway.
+        cls
+            The factory class that will be used to create the interaction.
+            By default, this is :py:class:`Interaction`. Should a custom
+            class be provided, it should be a subclass of :py:class:`Interaction`.
+
+        Returns
+        -------
+        Interaction
+            An instance :py:class:`Interaction` or the provided subclass.
+
+        Notes
+        -----
+        This is synchronous due to how slash commands are implemented.
```

Comment:
So it shows in a blue box like other notes
```suggestion
        .. note::

            This is synchronous due to how slash commands are implemented.
```